build/configs/artik05x/artik05x_download.sh

```diff
@@ -148,7 +148,7 @@ download()
 {
         parts_default=`grep -A 2 'config ARTIK05X_FLASH_PART_NAME' ${BOARD_KCONFIG} | sed -n 's/\tdefault "\(.*\)".*/\1/p'`
         parts2=${CONFIG_ARTIK05X_FLASH_PART_NAME:=${parts_default}}
-        parts=`echo $parts2 | sed "s/,/ /g"`
+        parts=${parts2//,/ }
```

Comment:
Please don't mix a way of indentation. Basically this file has a tab as an indentation.

Reply:
fixed